### ISSUE TITLE
Make the landing page for families who sign up but don't immediately have a study to take very friendly

### ISSUE THREAD

**Issue by mekline** [lookit/lookit-api#1065]
# TL;DR

Right now, a new family first sees the full set of studies, then has the disappointing experience of choosing a kid from the drop down menu and just seeing the below.  We should change this, in prep for wider advertisement of the site. 

<img width="1064" alt="Screen Shot 2023-01-20 at 2 00 21 PM" src="https://user-images.githubusercontent.com/6819829/213784163-0b343ade-1898-4461-9c1c-e7a937317239.png">

# Narrative

As a brand new user, it's disappointing to not have a study to take right away!  It's especially sad to see the big list of studies that disappears when I select the kid. Instead, I should get a friendly message telling me that I'll be emailed when there's a study available, and given something to click that *isn't* the old Children Helping Science platform! 

# Acceptance Criteria
A precise description of how/when this issue will be considered closed, e.g.:

- [ ] Decide the new flow for new participants who haven't provided a kid with any studies yet (IE something other than the vanilla "Find a study" link we currently show)
- [ ] Draft new text (Melissa!) for the Search page that makes sense for the empty return view, while still making sense for people reading it above a list of actual studies!
- [ ] Implement changes to new-participant workflow
- [ ] Implement changes to Search page

# Implementation Notes
Leave important notes/suggestions here for implementation (important files, suggested approaches, etc.)

**Comment by mekline** [comment]
An even nicer version of this would have the default view show the full set of studies for any child in the family, but this is out of scope!  See #1066 for that more extended feature, which I'm intentionally scoping *out* of this one.
